test/bats/azure.bats

```diff
@@ -121,6 +122,9 @@ setup() {
   result=$(kubectl get secret foosecret -o jsonpath="{.data.username}" | base64 -d)
   [[ "${result//$'\r'}" -eq "${SECRET_VALUE}" ]]
 
+  result=$(kubectl get secret foosecret -o jsonpath="{.metadata.labels[0]}")
```

Comment:
This command will error as labels is `map[string]interface{}`. It should be -

```suggestion
  result=$(kubectl get secret foosecret -o jsonpath="{.metadata.labels.environment}")
```

This will return the value of the label if exists